.gitignore

```diff
@@ -0,0 +1,2 @@
+doc/tags
+plugin/vinfo_doc_repo/
```

Comment:
Can't quite remember how this was supposed to work, but i think the plugin will fail if it can't find the `plugin/vinfo_doc_repo` directory, and it can't create it. Have you test that?